### PR TITLE
UMLEditor

### PR DESCRIPTION
**Pull request by 16aburnett**
Created the shell of the UMLEditor file. Implemented the REPL interface for the user to input commands.

### CODE REVIEW

**Review by adjusic — COMMENTED** [review]
Looks good!

File: code/UMLEditor.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# This file
+
+##########################################################################
+
+# from UMLModel import UMLModel
```
Comment (16aburnett) [inline code/UMLEditor.py:5]: Remember to un-comment this

**Review by carlishires — COMMENTED** [review]
I am getting this error when I try to run the UMLEditor.
umlEditorError

**Comment by carlishires** [comment]
I also got this error when I uncommented line 5.
umleditorerror2

**Comment by 16aburnett** [comment]
> I also got this error when I uncommented line 5.
> umleditorerror2

This branch only has the UMLEditor file and depends on the UMLModel class, so it is not expected to work pre-merge. If you comment out anything that mentions UMLModel then it should work fine. I should have left them commented out.

**Review by carlishires — CHANGES_REQUESTED** [review]
I found that "To be implemented" is printed out no matter what I enter into the command line.  I suggest that this should only be the case if the user inputs a valid command.
umlEditorKeysmash

**Comment by 16aburnett** [comment]
> I found that "To be implemented" is printed out no matter what I enter into the command line. I suggest that this should only be the case if the user inputs a valid command.
> umlEditorKeysmash

I believe david created a task where he planned on validating commands so i made it his job to implement validating commands in the execute function. 

(I know, this whole project has been so disorganized, we really should have assigned tasks better, but we should be able to move forward.)

**Review by DavitPetro — APPROVED** [review]
Looks good to start implementing code!

**Pull request merged**